scripts.js

```diff
@@ -1,6 +1,6 @@
-(function() {
-  var keys = [];
-  var sounds = [];
+{
+  let keys = {};
```

Comment:
`keys` needs to be global because it's called from both the `processData` function, and the `keydownEvent` function.